src/ui/layout/layout.svelte

```diff
@@ -17,6 +17,8 @@
     height: 100%;
     container-type: size;
     container-name: layout-query-container;
+    overflow-y: auto;
```

Comment:
overflow-y: auto needs to be set in here and in the child as well so that WebKit iOS does not clip the div to the parent height, and actually respects the overflow: auto of the parent (`rcb-ui-container.fullscreen`, the one with `position: fixed`)